.github/workflows/schedule.py

```diff
@@ -209,7 +221,32 @@ def create_issue(self, title_date: date) -> None:
                 assert set(labels) <= actual_labels, (labels, actual_labels)
                 actual_type = issue['type']['name']
                 assert type == actual_type, (type, actual_type)
-                log.info('Successfully created and verfied issue #%s', issue_number)
+                log.info('Successfully created and verified issue #%s', issue_number)
+                command = [
+                    'gh', 'project', 'item-add', azul_project_number,
+                    '--owner', 'DataBiosphere',
+                    '--url', url,
+                    '--format', 'json'
+                ]
+                log.info('Running %r …', command)
+                process = subprocess.run(command, check=True, stdout=subprocess.PIPE)
+                item = json.loads(process.stdout)
+                assert item['id'], item
+                priority = self.properties.get('_project_item_priority')
```

Comment:
I think using `.get` is a bad idea here, because if `priority` is `None` then line 241 will raise a `KeyError`.

```suggestion
                priority = self.properties['_project_item_priority']
```